coverage/index.html

```diff
@@ -81,93 +92,48 @@ <h2 id="roles_with_one_example_label">Roles with at Least One Guidance or Exampl
           <tr>
             <td><code>alertdialog</code></td>
             <td><abbr title="none" style="color: gray">-</abbr></td>
-            <td><a href="../examples/dialog-modal/alertdialog.html">Alert Dialog</a>
+            <td><a href="..\examples\dialog-modal\alertdialog.html">Alert Dialog</a>
 </td>
           </tr>
           <tr>
-            <td><code>article</code></td>
-            <td><abbr title="none" style="color: gray">-</abbr></td>
-            <td><a href="../examples/feed/feed.html">Feed</a>
-</td>
-          </tr>
-          <tr>
-            <td><code>banner</code></td>
-            <td><a href="../aria-practices.html#aria_lh_banner">Banner</a>
-</td>
-            <td><a href="../examples/landmarks/banner.html">Banner Landmark</a>
-</td>
-          </tr>
-          <tr>
-            <td><code>columnheader</code></td>
-            <td><abbr title="none" style="color: gray">-</abbr></td>
-            <td><a href="../examples/table/table.html">Table</a>
-</td>
-          </tr>
-          <tr>
-            <td><code>complementary</code></td>
-            <td><a href="../aria-practices.html#aria_lh_complementary">Complementary</a>
-</td>
-            <td><a href="../examples/landmarks/complementary.html">Complementary Landmark</a>
-</td>
-          </tr>
-          <tr>
-            <td><code>contentinfo</code></td>
-            <td><a href="../aria-practices.html#aria_lh_contentinfo">Contentinfo</a>
+            <td><code>checkbox</code></td>
+            <td><a href="../aria-practices.html#checkbox">Checkbox</a>
 </td>
-            <td><a href="../examples/landmarks/contentinfo.html">Contentinfo Landmark</a>
+            <td><a href="..\examples\checkbox\checkbox-2\checkbox-2.html">Checkbox  (Mixed-State)</a>
```

Comment:
it looks like the links are using backslash instead of forward-slash now, and I'm not sure why. They should probably use forward slash, but it'd be good to also make sure it won't change based on the OS it's run on.

Also, same as above -- I'm happy to poke at it in a bit, if you haven't already fixed it!

Reply:
@smhigley 
I think the problem is I generated the `index.html` file on my windows computer, so I have now used my Mac and it seems to have fixed this problem.